appium-dotnet-driver/Appium/Tizen/TizenDriver.cs

```diff
@@ -82,6 +85,29 @@ public TizenDriver(AppiumLocalService service, DesiredCapabilities desiredCapabi
         {
         }
 
+        public bool CompareToImages(Image origianalImage, Image compareImage)
+        {
+            MemoryStream originalImageStream = new MemoryStream();
+            MemoryStream compareImageStream = new MemoryStream();
+
+            origianalImage.Save(originalImageStream, origianalImage.RawFormat);
+            compareImage.Save(compareImageStream, compareImage.RawFormat);
+
+            return string.Equals(Convert.ToBase64String(originalImageStream.ToArray()), Convert.ToBase64String(compareImageStream.ToArray()));
+        }
+
+        public Image CropAndConvertImage(Screenshot screenshot, Rectangle rectangle)
```

Comment:
need to rename more clear something like `TransformScreenshot()` or `ConvertScreenshotToImage()`